### ISSUE TITLE
BUG: Versions with > or >= write an empty file to disk

### ISSUE THREAD

**Issue by elimumford** [raineorshine/npm-check-updates#1181]
- [ x ] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [ x ] I am using the latest version of `npm-check-updates`
- [ x ] I am using `node >= 14`

---

So, for the past few days, I've noticed an odd behavior of periodically ending up with a new un-tracked file in my git list, something like `16.11.51` or `18.7.9`.  I figured out that it was happening during `ncu`...  it appears to happen if the version ref is stated as `>=` or `>` for a package.  When ncu is processing those packages, it writes an empty file to disk with the filename of the suggested version.  In my testing so far, it only happens with the .ncurc file below, so some combination of those switches triggers it, as it does not appear to happen without an .ncurc file.

## Steps to Reproduce

In the .ncurrc and  Dependencies below, I have placed a minimum `.ncurc.json` and `package.json` file that should reproduce the issue.  Copy both the .ncurc.json and package.json to a directory and just run ncu in that location, and prior to being asked to select your updates it will have written 2 files to disk for the example version ref (@types/node and typescript) bugs. The @types/jest is there for reference as a suggested update that does not produce the behavior.

Of note, this is happening in a windows environment, so it is possibly environmental if unable to reproduce on macos/linux.  

.ncurc:

```json
{
  "dep": "prod,dev,bundle,optional,peer",
  "interactive": true,
  "peer": true,
  "target": "minor",
  "upgrade": true
}
```

Dependencies:

```json
{
  "name": "node-check-updates-bug",
  "version": "1.0.0",
  "description": "",
  "devDependencies": {
    "@types/jest": "^28.1.6",
    "@types/node": ">=16.11.49",
    "typescript": ">4.7.0"
  },
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}

```
## Current Behavior

Extra empty files.

## Expected Behavior

No extra files.

**Comment by raineorshine** [comment]
Hi, thanks so much for the bug report.

I can confirm that this does not occur on OSX. However, upon further investigation I have found that ncu is spawning `npm view typescript@>=4.8.2 peerDependencies` which is not right. It doesn't write to disk on OSX, but it does give an error of `4.8.2 not found`. The command should be `npm view typescript peerDependencies`.

I have published a fix to `v16.0.6`. Let me know if that fixes the issue.
